atd-vzd/triggers/fatality_insert.sql

```diff
@@ -0,0 +1,29 @@
+-- This function inserts a new record into the fatality table
+-- It is triggered by fatal insertions to the primaryperson or person tables
+CREATE OR REPLACE FUNCTION fatality_insert() 
+    RETURNS trigger
+    LANGUAGE plpgsql
+AS $function$
+    DECLARE
+    	year integer;
+        location text;
+    BEGIN
+    	SELECT TO_CHAR(crash_date,'yyyy')
+            FROM atd_txdot_crashes
+            WHERE crash_id = NEW.crash_id
+            INTO year;
+        SELECT CONCAT_WS(' ', rpt_block_num, rpt_street_pfx, rpt_street_name, '(', rpt_sec_block_num, rpt_sec_street_pfx, rpt_sec_street_name, ')')
+            FROM atd_txdot_crashes
+            WHERE crash_id = NEW.crash_id
+            INTO location;
```

Comment:
this creates the new location field that the VZ team wants in the fatality table